projects/aca-content/src/lib/components/toolbar/document-display-mode/document-display-mode.component.ts

```diff
@@ -47,15 +48,24 @@ import { AppStore, ToggleDocumentDisplayMode, getDocumentDisplayMode } from '@al
   encapsulation: ViewEncapsulation.None,
   host: { class: 'app-document-display-mode' }
 })
-export class DocumentDisplayModeComponent {
+export class DocumentDisplayModeComponent implements OnInit, OnDestroy {
   displayMode$: Observable<string>;
+  displayModeTitle: string;
 
-  constructor(private store: Store<AppStore>) {
-    this.displayMode$ = store.select(getDocumentDisplayMode);
+  onDestroy$ = new Subject<boolean>();
```

Comment:
new Subject<void>() will be enough

Reply:
Removed boolean from subject. New declaration is

`onDestroy$ = new Subject();`

Reply:
see my answer here, please correct: 
https://github.com/Alfresco/alfresco-content-app/pull/3227#discussion_r1206318128

Reply:
Updated declaration to be
```onDestroy$ = new Subject<void>();```